gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -67,6 +84,8 @@ void GazeboRosWheelSlip::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
               impl_->ros_node_->get_logger(),
               "New lateral slip compliance: %.3e", slip);
             this->SetSlipComplianceLateral(slip);
+          } else {
+            result.successful = false;
```

Comment:
We can also set the [reason](https://github.com/ros2/rcl_interfaces/blob/9a44202bbaf76e19fde21da0e90d8afbe89bd263/rcl_interfaces/msg/SetParametersResult.msg#L7) to something informative.